### PR TITLE
docs, debug: explain how to use dlv debugger 

### PR DESCRIPTION
**Pull request by nirdothan**
### What this PR does
Describe how to use a `delve` debugger with kubevirt go processes
Commits are incremented in steps of "depth", reviewers may choose to include only the first one or two commits 
replaces https://github.com/kubevirt/kubevirt/pull/13099

Before this PR:

After this PR:

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -71,5 +71,16 @@ journalctl -u kubelet
...
  - Add delve to the container
- - Start delve on a specific port ( `dlv attach <pid> --headless --listen=0.0.0.0:1234`)
+ - Start delve on a specific port ( `dlv attach <pid> --headless=true --api-version=2 --listen=:1234`)
  - Use kube-proxy to forward the port to your machine
+
+## Local execution of the kubevirt process
+Not all processses can easily be executed locally, since some of the processes require specific 
+dependencies that exist in the runtime pod and node
+For processes that are easily executed locally, such as `virt-controler`, the following program arguments
+can be passed to virt-controller `--kubeconfig /path/to/kubeconfig --leader-elect-lease-duration 99h` in order
```
Comment (xpivarc) [inline docs/debugging.md:82]: Maybe explain also the lease so somebody would not deem it as not important

**Review by xpivarc — COMMENTED** [review]
LGTM

**Comment by vladikr** [comment]
> LGTM

@xpivarc that's not an actual command ;)

**Comment by brianmcarey** [comment]
/override pull-kubevirt-e2e-k8s-1.31-sig-performance

Incorrectly triggered by status reconciler when lane was bumped.

**Pull request merged**